prisma/schema.prisma

```diff
@@ -37,3 +37,94 @@ model User {
   password  String
   createdAt DateTime @default(now())
 }
+
+//Real models
+model Role {
+  id            Int             @id @default(autoincrement())
+  name          String
+  Administrator Administrator[]
+  Barber        Barber[]
+  Customer      Customer[]
+}
+
+model State {
+  id            Int             @id @default(autoincrement())
+  name          String
+  Administrator Administrator[]
+  Barber        Barber[]
+  Customer      Customer[]
+}
+
+model Service {
+  id          Int           @id @default(autoincrement())
+  name        String
+  price       String
+  Appointment Appointment[]
+}
+
+model Administrator {
+  id              Int          @id @default(autoincrement())
+  name            String
+  document_number String
+  phone           String
+  birth_date      DateTime
+  url_img         String
+  email           String       @unique
+  stateId         Int
+  state           State        @relation(fields: stateId, references: id)
+  roleId          Int
+  role            Role         @relation(fields: [roleId], references: [id])
+  Engagement      Engagement[]
```

Comment:
Por qué el Engagement es un array?

Reply:
Haciendo uso de una extensión de prisma que me ayudaba a ver los errores, es necesario que al hacer relaciones foráneas con otras modelos, estos también tengan referencia a dichos modelos iniciales. Si no, se lanza el siguiente error. 

<img width="523" alt="image" src="https://user-images.githubusercontent.com/86166934/233830877-28c86014-1189-4993-a4b0-2b3728fe8fd0.png">

Reply:
Vale, y eso cómo justifica que sea un array?